java/client/src/main/java/glide/api/RedisClient.java

```diff
@@ -78,4 +87,118 @@ public CompletableFuture<Object> customCommand(String[] args) {
                 Command.builder().requestType(Command.RequestType.CUSTOM_COMMAND).arguments(args).build();
         return commandManager.submitNewCommand(command, BaseClient::handleObjectResponse);
     }
+
+    /**
+     * Ping the Redis server.
+     *
+     * @see <a href="https://redis.io/commands/ping/">redis.io</a> for details.
+     * @returns the String "PONG"
+     */
+    @Override
+    public CompletableFuture<String> ping() {
+        Command command = Command.builder().requestType(Command.RequestType.PING).build();
+        return commandManager.submitNewCommand(command, BaseClient::handleStringResponse);
+    }
+
+    /**
+     * Ping the Redis server.
+     *
+     * @see <a href="https://redis.io/commands/ping/">redis.io</a> for details.
+     * @param msg - the ping argument that will be returned.
+     * @returns return a copy of the argument.
+     */
+    @Override
+    public CompletableFuture<String> ping(String msg) {
+        Command command =
+                Command.builder()
+                        .requestType(Command.RequestType.PING)
+                        .arguments(new String[] {msg})
+                        .build();
+        return commandManager.submitNewCommand(command, BaseClient::handleStringResponse);
+    }
+
+    /**
+     * Get information and statistics about the Redis server. DEFAULT option is assumed
+     *
+     * @see <a href="https://redis.io/commands/info/">redis.io</a> for details.
+     * @return CompletableFuture with the response
+     */
+    @Override
+    public CompletableFuture<Map> info() {
+        Command command = Command.builder().requestType(Command.RequestType.INFO).build();
+        return commandManager.submitNewCommand(command, BaseClient::handleMapResponse);
+    }
+
+    /**
+     * Get information and statistics about the Redis server.
+     *
+     * @see <a href="https://redis.io/commands/info/">redis.io</a> for details.
+     * @param options - A list of InfoSection values specifying which sections of information to
+     *     retrieve. When no parameter is provided, the default option is assumed.
+     * @return CompletableFuture with the response
```

Comment:
```suggestion
     * @param options A list of <em>InfoSection</em> values specifying which sections of information to
     *     retrieve.
     * @return A <em>CompletableFuture</em> with the response
```

Optionally, refer to `info()`